preprocessor/webpack.js

```diff
@@ -1,32 +1,15 @@
 // Cypress webpack bundler adaptor
 // https://github.com/cypress-io/cypress-webpack-preprocessor
 const webpackPreprocessor = require('@cypress/webpack-preprocessor')
-const path = require('path')
 
-// default Cypress webpack options - good for basic projects
-const webpackOptions = {
-  resolve: {
-    extensions: ['.js', '.json', '.vue'],
-    alias: {
-      'cypress-vue-unit-test': path.join(__dirname, '..', 'src')
-    }
-  },
-  module: {
-    rules: [
-      {
-        test: /\.vue$/,
-        loader: 'vue-loader',
-        options: {
-          loaders: {
-            // you need to specify `i18n` loaders key with
-            // `vue-i18n-loader` (https://github.com/kazupon/vue-i18n-loader)
-            i18n: '@kazupon/vue-i18n-loader'
-          }
-        }
-      }
-    ]
-  }
-}
+const fw = require('find-webpack')
+const webpackOptions = fw.getWebpackOptions()
+
+// TODO: Figure out how to handle dynamic imports
+// Vue CLI's optimization and bundle splitting breaks Cypress
+// but it's necessary for <router-link> lazy loading...
+// and likely other dynamic imports
+delete webpackOptions.optimization
```

Comment:
you can probably use `cleanForCypress` method from https://github.com/bahmutov/find-webpack/blob/master/src/index.js - https://github.com/bahmutov/cypress-react-unit-test/blob/feature/cypress-mount-mode/plugins/cra-v3/file-preprocessor.js#L17

Reply:
Can't because I need the `plugins` section, which cleanForCypress deletes

Reply:
I have added a flag to clean method to mimic what you are doing and only remove problematic chunks plugins: https://github.com/bahmutov/find-webpack/blob/master/src/clean-for-cypress.js#L71

Maybe we can move your logic there and have a flag for Vue?

Reply:
Will refactor again after things go green